pkg/controller/advancedcronjob/advancedcronjob_controller.go

```diff
@@ -135,18 +131,32 @@ type ReconcileAdvancedCronJob struct {
 
 func (r *ReconcileAdvancedCronJob) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
+	klog.Infof("Running BroadcastCronJob job %s", req.Name)
```

Comment:
BroadcastCronJob -> AdvancedCronJob

Reply:
Done in commit https://github.com/openkruise/kruise/pull/461/commits/3d513814466ce219a7543280c4efc3a697a9fe78